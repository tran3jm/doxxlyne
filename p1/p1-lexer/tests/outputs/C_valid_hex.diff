--- outputs/C_valid_hex.txt	2022-09-09 03:34:50.317139213 -0400
+++ expected/C_valid_hex.txt	2022-09-08 01:56:15.007243957 -0400
@@ -1,4 +1,4 @@
-HEXLIT   [line -01]  0x340
-HEXLIT   [line -01]  0xD34
-HEXLIT   [line -01]  0xd45
-HEXLIT   [line -01]  0x9a2b
+HEXLIT   [line 001]  0x34
+HEXLIT   [line 002]  0xD34
+HEXLIT   [line 003]  0xd45
+HEXLIT   [line 004]  0x9a2b
\ No newline at end of file
