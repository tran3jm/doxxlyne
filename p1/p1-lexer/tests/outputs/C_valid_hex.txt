HEXLIT   [line 001]  0x340
DECLIT   [line 002]  0
ID       [line 002]  xD34
HEXLIT   [line 003]  0xd45
HEXLIT   [line 004]  0x9a2b
