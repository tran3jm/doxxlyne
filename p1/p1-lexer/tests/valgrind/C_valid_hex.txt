==3456314== Memcheck, a memory error detector
==3456314== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==3456314== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==3456314== Command: ../decaf inputs/valid_hex.decaf
==3456314== 
HEXLIT   [line 001]  0x340
HEXLIT   [line 002]  0xD34
HEXLIT   [line 003]  0xd45
HEXLIT   [line 004]  0x9a2b
==3456314== 
==3456314== HEAP SUMMARY:
==3456314==     in use at exit: 0 bytes in 0 blocks
==3456314==   total heap usage: 351 allocs, 351 frees, 53,968 bytes allocated
==3456314== 
==3456314== All heap blocks were freed -- no leaks are possible
==3456314== 
==3456314== For lists of detected and suppressed errors, rerun with: -s
==3456314== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
