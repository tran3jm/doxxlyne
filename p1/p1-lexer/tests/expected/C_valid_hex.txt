HEXLIT   [line 001]  0x340
HEXLIT   [line 002]  0xD34
HEXLIT   [line 003]  0xd45
HEXLIT   [line 004]  0x9a2b
